### PR TITLE
Update fabric-protos-go to v0.1.6

### PR DESCRIPTION
**Pull request by denyeart**
fabric-protos-go v0.1.6 has been released for Fabric v2.4.x.

Signed-off-by: David Enyeart <enyeart@us.ibm.com>

### CODE REVIEW

**Review by mbwhite — DISMISSED** [review]
LGTM

**Comment by C0rWin** [comment]
> LGTM

--- FAIL: TestNewPolicyErrorCase (0.00s)
    policy_test.go:106: 
        	Error Trace:	policy_test.go:106
        	Error:      	Error message not equal:
        	            	expected: "Error unmarshalling to SignaturePolicy: proto: common.SignaturePolicyEnvelope: illegal tag 0 (wire type 0)"
        	            	actual  : "Error unmarshalling to SignaturePolicy: proto: cannot parse invalid wire-format data"
        	Test:       	TestNewPolicyErrorCase
FAIL
coverage: 93.2% of statements

Seems like there is a failure which is related to the PR

File: go.mod [hunk 1]
```diff
@@ -30,7 +30,7 @@ require (
 	github.com/hyperledger/fabric-chaincode-go v0.0.0-20201119163726-f8ef75b17719
 	github.com/hyperledger/fabric-config v0.1.0
 	github.com/hyperledger/fabric-lib-go v1.0.0
-	github.com/hyperledger/fabric-protos-go v0.0.0-20220315113721-7dc293e117f7
```
Comment (pfi79) [inline go.mod:33]: fabric-protos-go to v0.1.6 has a comit 9c38fa002b9a, not 7dc293e117f7

File: go.mod [hunk 2]
```diff
@@ -20,7 +20,7 @@ require (
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/fsouza/go-dockerclient v1.7.0
 	github.com/go-kit/kit v0.9.0
-	github.com/golang/protobuf v1.3.3
+	github.com/golang/protobuf v1.5.0
```
Comment (pfi79) [inline go.mod:23]: it seems to me that `github.com/hyperledger/fabric-protos-go` should be returned `github.com/golang/protobuf v1.3.3`

Comment (jt-nti) [reply go.mod:23]: I think this error might be due to protobuf error texts being deliberately unstable...
https://developers.google.com/protocol-buffers/docs/reference/go/faq#unstable-errors

Comment (pfi79) [reply go.mod:23]: The main brunch and the tests on it were recently ported to protobuf 1.5.2 and backported to 2.5. It was not backported to 2.4.

**Comment by denyeart** [comment]
Thanks @pfi79 , yes I remember now that other changes are required to update protobuf. I'm going to close the PR for now since v2.5 will be available soon and will become the LTS release. Any v2.4 deployment should update to v2.5 at that time.

**Pull request closed without merging**